CHANGELOG.md

```diff
@@ -7,9 +7,11 @@
 
   - Major/Minor/Patch: This is an example changelog item, usually can be the commit message.
   - Append new items to make git merging easier.
+</details>
+
+## v0.48.2
   - Minor: Add Toggle Code button to all component examples in order to better accessibility QA
```

Comment:
Given that it's a minor change, I think we want this to be `v0.49.0`, but maybe @juanca would say otherwise